### PR TITLE
look for 'forge' or 'FTBserver' jar

### PR DESCRIPTION
**Pull request by colinrgodsey**
Same thing, think it's just their slightly modified fork. Or they just rename it to look cool.

### CODE REVIEW

**Comment by simon816** [comment]
Released in [0.6.1](https://github.com/simon816/SpongeBootstrap/releases/tag/v0.6.1)

**Comment by Twisted-Code** [comment]
thank you... I was hoping I wasn't going to have to rename the jar back manually or anything like that. This probably saves a lot of head scratching for users that can't even figure out it looks for something matching the "forge-universal.jar" pattern. while I would hope such users would either not bother trying to host a server, there's always that oddball "Instructions unclear. Got d*** stuck in server fan again"

**Comment by Twisted-Code** [comment]
actually, it still doesn't seem to be detecting it correctly. It still tells me "please make sure a forge jar exists". hmmm

**Comment by colinrgodsey** [comment]
oh no, I put a PR up for the wrong version lol

File: src/main/java/com/simon816/sponge/bootstrap/Bootstrap.java [hunk 1]
```diff
@@ -84,7 +84,8 @@ private static void findAndLoadJars() {
             @Override
             public boolean accept(File pathname) {
                 String fn = pathname.getName().toLowerCase();
-                return fn.endsWith(".jar") && fn.contains("forge") && fn.contains("-universal") && supportedVersion(fn);
+                return fn.endsWith(".jar") && (fn.contains("forge") || fn.contains("FTBserver"))
```
Comment (colinrgodsey) [inline src/main/java/com/simon816/sponge/bootstrap/Bootstrap.java:87]: gah. FTBserver here needs to be lower case, forgot I copied my checkout elsewhere on my machine

**Comment by colinrgodsey** [comment]
https://github.com/simon816/SpongeBootstrap/pull/13/files#diff-b18bdc3c193fe998714f27eaa86d38a4R87

**Comment by simon816** [comment]
@macks2008 should be fixed now in [0.6.2](https://github.com/simon816/SpongeBootstrap/releases/tag/v0.6.2)

**Pull request merged**